### ISSUE TITLE
[Feature] Handle token in the configuration command

### ISSUE THREAD

**Issue by ajeans** [jfrog/jfrog-cli#107]
Right now, "jfrog rt config" handles credentials:
* user/password, with an additional flag to apply encryption (or not)
* apikey

However, when using access tokens with jfrog cli, the command to configure the server with the token is complicated:

`jfrog rt config tokenServerID --url=http://artifactoryserver:8081/artifactory --user=tokenname --password=$MYLONGTOKEN --enc-password=false --interactive=false`

It is not intuitive that an access token has to be declared as an unencrypted password. It is also annoying that a tokenname has to be declared when it is not necessary with the "Authentication Bearer" option.
Would it be possible to get a new `--token` option added to the config command, and have all commands authenticate with the token as "Authentication Bearer"?

**Comment by tdillon** [comment]
I agree that using an access token to authenticate with the CLI is not intuitive.  I didn't know it was possible until I found this issue (thanks @ajeans).  At the very least the CLI documentation should be updated to give examples/explanation of using access tokens with the CLI.

**Comment by eyalbe4** [comment]
@tdillon and @ajeans - 
You may be already aware of this, but Access Tokens supported has been added to JFrog CLI recently.
Please let su know what you think about the new functionality.
